docs/cli-reference/ks_env_targets.md

```diff
@@ -4,24 +4,43 @@ Set module targets for an environment
 
 ### Synopsis
 
-targets
+
+The `targets` command selects one or more modules to be select by an
+environment. The default environment target is the root path, `/`.
```

Comment:
Root path -> root module

Reply:
Should we mention that when changing targets, `/` will no longer be included implicitly?

Reply:
Another potential callout - setting targets overwrites the entire list - they should always specify the full list of targets when updating.